test/check-openshift

```diff
@@ -89,4 +90,50 @@ class TestOpenshift(MachineCase):
         b.wait_present("svg line.DeploymentConfigReplicationController")
         b.wait_present("svg line.RouteService")
 
+    def testDelete(self):
+        m = self.machine
+        b = self.browser
+        b.wait_timeout(120)
+
+        self.login_and_go("/kubernetes")
+        b.wait_present("#service-list")
+
+        self.openshift.execute("oc create -f /tmp/mock-app-openshift.json")
+        b.wait_in_text("#service-list", "mock")
+
+        b.click("a[href='#/list']")
+        b.wait_present("#content .details-listing")
+        b.wait_present("#routes")
+        b.wait_present("#deployment-configs")
+
+        b.wait_present(".details-listing tbody[data-id='deploymentconfigs/default/frontend'] th")
+        b.wait_present(".details-listing tbody[data-id='routes/default/mock'] th")
+        self.assertEqual(b.is_present(".details-listing tbody[data-id='routes/default/mock'] th"), True)
+        self.assertEqual(b.is_present(".details-listing tbody[data-id='deploymentconfigs/default/frontend'] th"), True)
+
+        b.click(".details-listing tbody[data-id='routes/default/mock'] th")
+        b.click(".route-delete")
```

Comment:
It's not visible. See the screenshot: http://files.cockpit-project.org/hubbot/571b11dcd86fd449a8aaa3f6133acdcbe6d15812_f22_x86-64/test/check-openshift-testDelete-FAIL.png

You need to click on the row to open it. See check-kubernetes where we click on ```tr.listing-item```